mtools/test/test_mloginfo.py

```diff
@@ -1,14 +1,24 @@
+import io
 import os
 import re
 import sys
 from datetime import timedelta, date
 from random import randrange
 
+import pytest
+
 import mtools
 from mtools.mloginfo.mloginfo import MLogInfoTool
 from mtools.util.logfile import LogFile
 
 
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
Nice monkeypatch. Ideally we should be unit testing output of `run()` rather than grubbing from stdout, but we can fix that later.